api/v1beta2/kyma_types.go

```diff
@@ -388,14 +388,6 @@ func (kyma *Kyma) AllModulesReady() bool {
 	return true
 }
 
-func (kyma *Kyma) HasSyncLabelEnabled() bool {
-	if sync, found := kyma.Labels[shared.SyncLabel]; found {
```

Comment:
How about `shared.SyncLabel`, should be removable right?

Reply:
Can't remove `shared.SyncLabel` in this PR, as module templates still depend on it. It still needs to be clarified (third AC of this issue from Jan).

Reply:
Alright, then let's just keep this PR open for a bit until this is clarified in a couple of days.